Release/v0.7.1 (#21)
fivetran/dbt_ga4_export: CHANGELOG.md

# dbt_ga4_export v0.7.1

[PR #21](https://github.com/fivetran/dbt_ga4_export/pull/21) includes the following updates:

## Bug Fixes

> - Wraps `user_pseudo_id` in a `coalesce` when building `session_id` in `int_ga4_export__derived_event_fields` to prevent null test failures.

Reviewer: ```suggestion
- Prevents null `session_id` test failures in `ga4_export__sessions_enhanced` when `user_pseudo_id` is null.
```

> - Hard casts `param_session_engaged` in `stg_ga4_export__event` to map string values to booleans to avoid data type failures.

Reviewer: ```suggestion
- Ensures proper mapping of `param_session_engaged` string values to booleans in `stg_ga4_export__event`
```

> - Casts both `param_ga_session_number` and `derived_session_index` to strings individually within the `session_number` coalesce in `int_ga4_export__derived_event_fields`, to avoid data type collisions.

Reviewer: ```suggestion
- Casts `param_ga_session_number` and `derived_session_index` as strings before coalescing as `session_number` in `int_ga4_export__derived_event_fields` to avoid data type collisions.
```

Reviewer: Also may want to note that session_number is definitely a string for everyone now
